### PR TITLE
description added

### PR DESCRIPTION
**Pull request by pal3635**
added more about standalone java script engines and their descriptions. added more description to the original java script definition to make it better understndable

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/javascript/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: JavaScript
 ## JavaScript
 JavaScript is the most widely used scripting language on Earth. And it has the largest library ecosystem of any programming language.
 
-JavaScript is the core language of the web, and the only programming language that can run in all major web browsers.
+JavaScript is the core language of the web, and the only programming language that can run in all major web browsers. It is used to add more creative and entertaining content to the site by introducting animation and visul effects. It helps your website look more lively.
```
Comment (amandeepmittal) [inline guide/english/javascript/index.md]: I do not think this ` It is used to add more creative and entertaining content to the site by introducting animation and visul effects. It helps your website look more lively.` adds any value to existing content. Please remove it.

File: guide/english/javascript/index.md [hunk 2]
```diff
@@ -15,20 +15,30 @@ If you want to learn more about the JavaScript language, and why it's so widely
 or watch this [inspiring video from Preethi Kasireddy](https://www.youtube.com/watch?v=VqiEhZYmvKk).
 
 ## Standalone JavaScript engines
-Mozilla's SpiderMonkey, the first JavaScript engine ever written, currently used in Mozilla Firefox.
+-Mozilla's SpiderMonkey, the first JavaScript engine ever written, currently used in Mozilla Firefox.
```
Comment (amandeepmittal) [inline guide/english/javascript/index.md]: All the list items are not properly formatted according to Markdown as previously they were. Please format them correctly.

File: guide/english/javascript/index.md [hunk 3]
```diff
@@ -15,20 +15,30 @@ If you want to learn more about the JavaScript language, and why it's so widely
...
 V8, Google's JavaScript engine, used in Google Chrome.
 
-Google Apps Script, a cloud-based/server-side interpreter that provides programmatic "macro-like" control of Google Apps services and documents.
+-Google Apps Script, a cloud-based/server-side interpreter that provides programmatic "macro-like" control of Google Apps services and - documents.
```
Comment (amandeepmittal) [inline guide/english/javascript/index.md]: There should not be any hyphen `-` in ` and - documents.`

File: guide/english/javascript/index.md [hunk 4]
```diff
@@ -15,20 +15,30 @@ If you want to learn more about the JavaScript language, and why it's so widely
...
 
-Node.js, built on top of V8, a platform which enables server-side applications to be written in JavaScript.
+-Node.js, built on top of V8, a platform which enables server-side applications to be written in JavaScript.
```
Comment (amandeepmittal) [inline guide/english/javascript/index.md]: `V8, a platform` should be `V8, is a platform`

File: guide/english/javascript/index.md [hunk 5]
```diff
@@ -15,20 +15,30 @@ If you want to learn more about the JavaScript language, and why it's so widely
...
 
-Windows includes JScript, a JavaScript variant in Windows Script Host.
+-Windows includes JScript, a JavaScript variant in Windows Script Host.
 
-Chakra, a fork of Jscript, is developed by Microsoft and used in their Edge browser.
+-Chakra, a fork of Jscript, is developed by Microsoft and used in their Edge browser.
 
-Mozilla also offers Rhino, an implementation of JavaScript built in Java, typically embedded into Java applications to provide scripting to end users.
+-Mozilla also offers Rhino, an implementation of JavaScript built in Java, typically embedded into Java applications to provide        scripting to end users.
```
Comment (amandeepmittal) [inline guide/english/javascript/index.md]: No need for so much extra spacing: `provide        scripting`

**Review by amandeepmittal — CHANGES_REQUESTED** [review]

File: guide/english/javascript/index.md (hunk 1 above)
Comment (pal3635) [reply guide/english/javascript/index.md]: Thanks for the feedback. I will change it accordingly

**Review by Manish-Giri — COMMENTED** [review]
Hi ,

I am closing this PR as there has been no response from your end for a while.

Feel free to re-open this PR if you are able to address the suggested changes.

**Pull request closed without merging**